lib/business/calendar.rb

```diff
@@ -54,9 +53,10 @@ def self.load_cached(calendar)
 
     DAY_NAMES = %( mon tue wed thu fri sat sun )
 
-    attr_reader :holidays, :working_days, :extra_working_dates
+    attr_reader :name, :holidays, :working_days, :extra_working_dates
 
-    def initialize(config)
+    def initialize(name, config)
```

Comment:
As above, let's convert this to have explicit nil kwargs, something along the lines of:

```suggestion
    def initialize(name:, extra_working_dates: nil, working_days: nil, holidays: nil)
```